bbl_screen-patch/kexec_ui/printerui/qml/dialog/KexecDialog.qml

```diff
@@ -16,34 +17,78 @@ Item {
         return xhr.status == 200;
     }())
     property var countdown: 10
+    property string currentMode: (x1pName !== "") ? "ota" : "default"
```

Comment:
I think I would probably spell this as a bool rather than as a string, given that it has only two settings, though I guess using this as a key for startupStrings makes some sense